src/command_modules/azure-cli-rdbms/tests/test_rdbms_commands.py

```diff
@@ -0,0 +1,377 @@
+                 checks=[
+                     JMESPathCheck('name', firewall_rule_1),
+                     JMESPathCheck('resourceGroup', rg),
+                     JMESPathCheck('startIpAddress', start_ip_address_2),
+                     JMESPathCheck('endIpAddress', end_ip_address_2)])
+
+        self.cmd('{} server firewall-rule update --name {} -g {} --server {} '
+                 '--start-ip-address {}'
+                 .format(engine, firewall_rule_1, rg, server,
+                         start_ip_address_1),
+                 checks=[
+                     JMESPathCheck('name', firewall_rule_1),
+                     JMESPathCheck('resourceGroup', rg),
+                     JMESPathCheck('startIpAddress', start_ip_address_1),
+                     JMESPathCheck('endIpAddress', end_ip_address_2)])
+
+        self.cmd('{} server firewall-rule update -n {} -g {} -s {} '
+                 '--end-ip-address {}'
+                 .format(engine, firewall_rule_1, rg, server,
+                         end_ip_address_1),
+                 checks=[
+                     JMESPathCheck('name', firewall_rule_1),
+                     JMESPathCheck('resourceGroup', rg),
+                     JMESPathCheck('startIpAddress', start_ip_address_1),
+                     JMESPathCheck('endIpAddress', end_ip_address_1)])
+
+        # test firewall-rule create another rule
+        self.cmd('{} server firewall-rule create --name {} -g {} --server {} '
+                 '--start-ip-address {} --end-ip-address {}'
+                 .format(engine, firewall_rule_2, rg, server,
+                         start_ip_address_2, end_ip_address_2),
+                 checks=[
+                     JMESPathCheck('name', firewall_rule_2),
+                     JMESPathCheck('resourceGroup', rg),
+                     JMESPathCheck('startIpAddress', start_ip_address_2),
+                     JMESPathCheck('endIpAddress', end_ip_address_2)])
+
+        # test firewall-rule list
+        self.cmd('{} server firewall-rule list -g {} -s {}'
+                 .format(engine, rg, server), checks=[JMESPathCheck('length(@)', 2)])
+
+        self.cmd('{} server firewall-rule delete --name {} -g {} --server {} --yes'
+                 .format(engine, firewall_rule_1, rg, server), checks=NoneCheck())
+        self.cmd('{} server firewall-rule list -g {} --server {}'
+                 .format(engine, rg, server), checks=[JMESPathCheck('length(@)', 1)])
+        self.cmd('{} server firewall-rule delete -n {} -g {} -s {} --yes'
+                 .format(engine, firewall_rule_2, rg, server), checks=NoneCheck())
+        self.cmd('{} server firewall-rule list -g {} --server {}'
+                 .format(engine, rg, server), checks=[NoneCheck()])
+
+    def _test_db_mgmt(self, resource_group, server, engine):
+        # database_name = "cliautomationdb01"
+        # charset = 'utf8'
+        # collation = 'utf8_general_ci' if engine == 'mysql' else 'English_United States.1252'
```

Comment:
Want to keep them here because it was initially required, but to keep consistent with portal, the commands was temporarily removed, in case we need to re-open these commands later.